### PR TITLE
Add progressbar and checks to roadmap md

### LINKED ISSUE #132: Beautify roadmap md
@dholbach and I plan to add a [progress bar](https://facelessuser.github.io/pymdown-extensions/extensions/progressbar/) and change the bullets to checkmarks. Checked/finished items will be in grey.

### PR DESCRIPTION
**Pull request by bia**
Resolves #132

Screenshot 2020-07-29 at 21 08 48

### CODE REVIEW

File: docs/roadmap/index.md [hunk 1]
```diff
@@ -16,70 +18,70 @@ and [FluxCloud](https://github.com/justinbarrick/fluxcloud) notifications.
...
-- ~~Offer a dedicated component for forwarding events to external messaging platforms~~
+- [ ]  Offer an in-place migration tool for those that are using Flux in read-only mode to synchronize plain manifests
+- [ ]  Offer a migration guide for those that are using Flux in read-only mode to synchronize Kustomize overlays
+- [x]  <span style="color:grey">Offer a dedicated component for forwarding events to external messaging platforms</span>
 
 Non-Goals
 
-- Migrate users that are using Flux to run custom scripts with `flux.yaml`
-- Automate the migration of `flux.yaml` kustomize users
+- [ ]  Migrate users that are using Flux to run custom scripts with `flux.yaml`
```
Comment (hiddeco) [inline docs/roadmap/index.md:27]: Shouldn't this be a normal list? Given these are non-goals and are thus never going to be fulfilled.

**Comment by stefanprodan** [comment]
@bia can we have a progressbar per milestone?

File: docs/roadmap/index.md (hunk 1 above)
Comment (stefanprodan) [reply docs/roadmap/index.md:27]: I think only the tasks should have checklists

Comment (bia) [reply docs/roadmap/index.md:27]: Sounds good!
How about this then?
Screenshot 2020-07-30 at 12 07 32

**Comment by bia** [comment]
>@bia can we have a progressbar per milestone?

@stefanprodan You mean like this?
Screenshot 2020-07-30 at 12 22 41

**Comment by stefanprodan** [comment]
I would place the progressbar after each milestone title instead of listing them twice.

**Comment by bia** [comment]
Great idea!
Screenshot 2020-07-30 at 12 59 33

File: docs/roadmap/index.md (hunk 1 above)
Comment (hiddeco) [reply docs/roadmap/index.md]: Slightly triggered by the misalignment of the second line of text next to the :heavy_check_mark:, other than that :ok_hand:

**Comment by bia** [comment]
Fixed. 🙂 
Screenshot 2020-07-30 at 21 57 05

**Comment by dholbach** [comment]
Beautiful work! <3

**Pull request merged**